pom.xml

```diff
@@ -56,7 +56,7 @@
     <json-simple.version>1.1.1</json-simple.version>
     <commons-lang.version>2.6</commons-lang.version>
     <rxjava.version>2.2.3</rxjava.version>
-    <kafka-clients.version>1.0.2</kafka-clients.version>
+    <kafka-clients.version>3.4.0</kafka-clients.version>
```

Comment:
If this version is already in the parent pom, why do we still need it here?  As long as this pom and others are descended from the parent, they don't need to set it again unless the value must be different from the parent pom.
I think you can delete this line entirely.